marketplace/optimizely/extension.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "optimizely",
```

Comment:
Is this augmenting or replacing the existing [Optimizely Audiences](https://www.contentful.com/developers/marketplace/optimizely-audiences/) extension?

Either way I'd suggest changing the ID (and the name of the directory) to `optimizely-audiences` so that additional Optimizely extensions can be developed. (E.g. Optimizely themselves have created `Optimizely Events`, `Optimizely Audiences` and `Optimizely results`.)